presto-docs/src/main/sphinx/sql/describe-output.rst

```diff
@@ -0,0 +1,85 @@
+===============
+DESCRIBE OUTPUT
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    DESCRIBE OUTPUT statement_name
+
+Description
+-----------
+
+Describes the output columns of a prepared statement.  It returns a table
+containing metadata for output columns of a prepared statement. For each
+output column, the column name (or alias), catalog, schema, table, type,
+type size in bytes, and a boolean indicating if the column is aliased is
+returned.
+
+Examples
+--------
+
+Prepare and describe a query with four output columns:
+
+.. code-block:: sql
+
+    PREPARE my_select1 FROM
+    SELECT * FROM nation
+
+.. code-block:: sql
+
+   DESCRIBE OUTPUT my_select1;
+
+.. code-block:: none
+
+         Column Name | Catalog | Schema | Table  |  Type   | Type Size | Aliased
+        -------------+---------+--------+--------+---------+-----------+---------
+         nationkey   | tpch    | sf1    | nation | bigint  |         8 | false
+         name        | tpch    | sf1    | nation | varchar |         0 | false
+         regionkey   | tpch    | sf1    | nation | bigint  |         8 | false
+         comment     | tpch    | sf1    | nation | varchar |         0 | false
+        (4 rows)
+
+Prepare and describe a query whose output columns are expressions:
+
+.. code-block:: sql
+
+   PREPARE my_select2 FROM
+   SELECT count(*) as my_count, 1+2 FROM nation
+
+.. code-block:: sql
+
+    DESCRIBE OUTPUT my_select2;
+
+.. code-block:: none
+
+         Column Name | Catalog | Schema | Table |  Type  | Type Size | Aliased
+        -------------+---------+--------+-------+--------+-----------+---------
+         my_count    |         |        |       | bigint |         8 | true
```

Comment:
Are these blanks even correct?

Reply:
yup because the column is a computed value it's not associated with any catalog, schema, or table.